Library/OcAfterBootCompatLib/ServiceOverrides.c

```diff
@@ -964,10 +966,14 @@ OcExitBootServices (
   // For non-macOS operating systems return directly.
   //
   if (BootCompat->ServiceState.AppleBootNestedCount == 0) {
-    return BootCompat->ServicePtrs.ExitBootServices (
+    Status = BootCompat->ServicePtrs.ExitBootServices (
       ImageHandle,
       MapKey
       );
+    if (EFI_ERROR (Status)) {
+      DEBUG ((DEBUG_WARN, "OCABC: OcExitBootServices (%amacOS) - %r\n", "non-", Status));
```

Comment:
You are not allowed to print within/after ExitBootServices. Even on failure.

Reply:
https://github.com/acidanthera/OpenCorePkg/blob/master/Library/OcMainLib/OpenCoreUefiInOut.c#L63-L67

Reply:
Actually, I guess the above is not (strictly) within or after, so is different?

Reply:
Actually, again, no - the above is still after the memory map has been taken - so not entirely clear of the difference... ... I suppose in this case here, the memory map has been taken, and the debug lib might have been torn down? (Probably the other comment is sightly misleading still, though? Or could do with a new comment - maybe?? - where I tried to add some code?)

Reply:
I won't push this - ofc! - and I'll offer sightly updated comments in as separate commit and see what you think.

Reply:
The example is different, as the print is almost impossible to appear, and is a bug. Here the print will frequently trigger on many machines and will bork them.